financialaid/constants.py

```diff
@@ -0,0 +1,224 @@
+"""
+Constants for financialaid
+"""
+
+
+DEFAULT_INCOME_THRESHOLD = 100000
+COUNTRY_INCOME_THRESHOLDS = {
```

Comment:
What's the source for the threshold data here and in the CSV? I am getting some disprepancies with the CSV files I got in an email from Anna. I imported those CSVs, removed the blank lines, and converted these country codes to three letter country codes.

    In [78]: set(expected.keys()) - set(thresholds.keys())
    Out[78]: {'ADO', 'CHI', 'IMY', 'KSV', 'ROM', 'TMP', 'WBG', 'ZAR'}
    
    In [79]: set(thresholds.keys()) - set(expected.keys())
    Out[79]: {'AND', 'COD', 'IMN', 'PSE', 'ROU', 'TLS', 'XKX'}